.github/CONTRIBUTING.rst

```diff
@@ -0,0 +1,53 @@
+#####################################
+Documentation contribution guidelines
+#####################################
+
+External contributions
+======================
+
+This is an open source project and LSST welcomes contributions from the community.
+Feel free to create a GitHub issue to report a problem or even submit a pull request.
+Issues and pull requests are triaged into LSST's ticketing system, and we'll be able to resolve those issues or merge those pull requests for you.
+
+Project contributors
+====================
+
+If you're a member of the LSST Project, and have push access to this repository, please follow `LSST DM's Development Workflow <https://developer.lsst.io/work/flow.html>`__, particularly including the branching conventions.
+
+Content guidelines
+==================
+
+Most of this project's content is written in reStructuredText.
+Please see the `DM ReStructuredText Style Guide <https://developer.lsst.io/restructuredtext/style.html>`__ for information on how to format things like headers, lists, tables, images, and code samples.
+
+Please write one sentence per line (as opposed to hard-wrapping text to a specific line width).
+This makes Git diffs and pull requests easier to use.
```

Comment:
I think I usually see `git` spelled in all lower case.

Reply:
Sorry, never mind.  You are not referring to the command line program but to the system as a proper noun.  Disregard this comment.

Reply:
GitHub makes this distinction in their documentation too. Super tricky, I know :)